### ISSUE TITLE
Command to programmatically get the generated web action urls

### ISSUE THREAD

**Issue by alexkli** [adobe/aio-cli-plugin-app#183]
### Motivation

In a CI or other developer tooling one might want to programmatically get the url of the web actions.

However, because the openwhisk package name is dynamically generated by `aio app` (using folder name, module version) and the action name could also be subject to certain dynamic adjustments, it is difficult to use `aio runtime action get -r` since one does not know the full name of the deployed action.

### Feature request

I suggest these commands:

1. get url for an action. `<action>` is the name of the action in the `manifest.yml`:
    ```
    aio app get-url <action>
    ```
   outputs just the url so it can be easily used in shell scripts
2. get url for all actions
    ```
    aio app get-url
    ```
   outputs one url on each line

### Workaround

As workaround, I added this shell script, with the package name logic AFAICS (not sure if always correct):
```
export ACTION_NAME=worker
export ACTION_PKG=$(node -e 'p = require("./package.json"); console.log(`${p.name}-${p.version}`)')
export WEB_ACTION_URL=https://${AIO_RUNTIME_NAMESPACE}.adobeioruntime.net/api/v1/web/${ACTION_PKG}/${ACTION_NAME}
```

**Comment by meryllblanchet** [comment]
Thanks for the proposal @alexkli ! 
I'd like to add some `--cdn` option to also output the URL of the actions behind the CDN for SPA use-cases.

**Comment by moritzraho** [comment]
fix by #186
